.github/workflows/integration.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           distribution: 'adopt'
 
       - name: Setup Docker
-        uses: docker-practice/actions-setup-docker@v1
+        uses: docker-practice/actions-setup-docker@master
```

Comment:
These "Setup Docker" steps can be completely removed (I have it in #21 also, but can be done here first). The ubuntu-latest VMs have it already.